source/install.txt

```diff
@@ -91,21 +91,22 @@ Select the appropriate tab for your operating system:
       Considerations for Ubuntu
       ~~~~~~~~~~~~~~~~~~~~~~~~~
 
-      - ``mongosh`` is available as a
+      - :binary:`mongosh` is available as a
         :abbr:`PPA (Personal Package Archive)` for Ubuntu 20.04 (Focal)
-        and Ubuntu 18.04 (Bionic).
+        and Ubuntu 18.04 (Bionic). If you are installing on Ubuntu
+        20.04 (Focal) or Ubuntu 18.04  (Bionic), use the procedure in
+        the ``.deb`` tab.
```

Comment:
It seems redundant to tell users which tab to click both in the "Considerations for Ubuntu" section and in the "Procedure" section. Maybe we can clean this up a bit?